code-challenges.test.js

```diff
@@ -13,230 +10,50 @@ const { describe } = require("yargs")
-//Use the "decribe()" convention
-//yarn add jest
-//yarn jest
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-describe("anItem", () => {
-   it("returns a number", () => {
-     expect(thisItem("yes")).toEqual("15 is divisible by three")
-   })
- })
- const object2 = { number: 0 }
- // Expected output: "0 is divisible by three"
- describe("thisOtherItem", () => {
-    it("returns a number", () => {
-      expect(object2("yes")).toEqual("0 is divisible by three")
-    })
-  })
- const object3 = { number: -7 }
- // Expected output: "-7 is not divisible by three"
- describe("lastItem", () => {
-    it("returns a number", () => {
-      expect(object1("yes")).toEqual("-7 is is not divisible by three")
-    })
-  })
-
-// I wrote a test for each of the statements above, with expect statements but each of them failed the test with the output in terminal (look below). Instead of getting "gppd failure" of ReferenceError: greeter is not defined, terminal told me that "object1" is not a function. 
-//orginally, inside of the describe statement I had "object 1" "object 2" and "object 3". But after the test failed, I changed it to "anItem" "thisOtherItem" and "lastItem" because I figured that it needed to be different from the expect statement. Both the describe and expect being the same is the reason I think the jest result told me that it was not a function. By the time I ran the test again, the Jest test suite failed to run.
-
-// FAIL  ./code-challenges.test.js
-  //object1
- // ✕ returns a number
-
-●// object1 › returns a number
-
- // TypeError: object1 is not a function
-
-   // 40 | describe("object1", () => {
-   // 41 |    it("returns a number", () => {
-  //> 42 |      expect(object1("yes")).toEqual("15 is divisible by three")
-   //   |             ^
-   // 43 |    })
-  // 44 |  })
-   // 45 |
-   // at Object.object1 (code-challenges.test.js:42:13)
 
+const numbersArray1 = [6, 7, 8, 9, 10]
+// Expected output: [18, 21, 24, 27, 30]
+const numbersArray2 = [24, 27, 30, 33, 36]
+// Expected output: [72, 81, 90, 99, 108]
 
 // b) Create the function that makes the test pass.
 
-// function 
-//PSEUDO CODE
-//Input: Self-invoked
-//Output: string "15 is divisible by three"
-//Declare a function
-
-const object1 = (answer) => {
-   if (answer [15] % 3 == 0) {
-      return "15 is divisible by three"
-   }
-
-
-// function 
-//PSEUDO CODE
-//Input: Self-invoked
-//Output: "0 is divisible by three"
-//Declare a function
+// --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
+// a) Create a test with expect statements for each of the variables provided.
 
-const object2 = (answer) => {
-   if (answer [0] % 3 == 0) {
-      return "0 is divisible by three"
-   } 
-
-
-// function 
-//PSEUDO CODE
-//Input: Self-invoked
-//Output: "-7 is not divisible by three"
-//Declare a function
-
-
-const object3= (answer) => {
```

Comment:
Added to above comment: So these given variables would be the argument for the function you make with these conditionals inside of it.